src/views/OrganizationPageView/LocalComponents/PluginComponent/PluginComponent.tsx

```diff
@@ -66,6 +80,38 @@ const PluginComponent = React.forwardRef((props: PluginComponentPropsStyled, ref
                     </Grid>
                 </Grid>
             </Grid>
+        )
+    }
+    else{
+        item=(
```

Comment:
Where are spaces? Tap ctrl + alt + L or hook thi commant to save event in webstorm!